web/readme.md

```diff
@@ -1,14 +1,22 @@
 Samples for [Flutter for web](https://flutter.dev/web).
 
-#### Building Samples code
-Set up `webdev` for first time as below:
-```
-$ pub global activate webdev
-$ pub get
-```
+## See built samples
 
-Build and run code locally at `http://localhost:8080`
-```
+All of the samples are at [flutter.github.io/samples](https://flutter.github.io/samples/).
```

Comment:
Consider "Compiled versions of the samples are hosted at...."

Reply:
👍